releases/v1.3.5.toml

```diff
@@ -0,0 +1,27 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release
+previous = "v1.3.4"
+
+pre_release = false
+
+preface = """\
+The fifth patch release for `containerd` 1.3 includes bug fixes.
```

Comment:
What is the most notable fix that makes a release necessary. I think it is good to highlight that here.

Reply:
updated